content/en/docs/tasks/tools/install-kubectl-linux.md

```diff
@@ -30,28 +30,45 @@ The following methods exist for installing kubectl on Linux:
 
 1. Download the latest release with the command:
 
-   ```bash
+   {{< tabs name="download_binary_linux" >}}
+   {{< tab name="x86-64" codelang="bash" >}}
```

Comment:
In other pages, I saw the shortcode written as `{{% tab %}}`, but I'm not sure which one is correct or why there are two flavors of the same thing.

Reply:
@tengqm, I've used the Tabs format that's described in the Hugo Documentation: https://kubernetes.io/docs/contribute/style/hugo-shortcodes/#tabs
If you click on the **Deploy Preview** link above, you will see that it renders correctly.

Reply:
The shortcode is written to work with `{{< tab >}}` because of a historical detail about how we migrated to Hugo.

Reply:
Which is to say this is laid out correctly. Thanks for the context @sftim.

/approve